composer.json

```diff
@@ -1,12 +1,12 @@
 {
-    "name": "wulfheart/pretty_routes",
+    "name": "elsayed85/pretty_routes",
```

Comment:
The vendor of that package doesn't change.

Reply:
done